src/primitives/boolean.js

```diff
@@ -0,0 +1,8 @@
+export default class MicrostateBoolean {
```

Comment:
Let's call this `BooleanState` since `Microstate` is the less significant thing about them and `Boolean` is the more significant thing. Same applies for the other primitives.

Reply:
Sounds good